content/en/docs/build/configuration.md

```diff
@@ -2,39 +2,43 @@
 title: "Configuration"
 draft: false
 ---
-
 <!--
 Copyright The Shipwright Contributors
 
 SPDX-License-Identifier: Apache-2.0
 -->
-
 ## Controller Settings
 
-The controller is installed into Kubernetes with reasonable defaults. However, there are some settings that can be overridden using environment variables in [`controller.yaml`](https://github.com/shipwright-io/build/blob/v0.10.0/deploy/500-controller.yaml).
+The controller is installed into Kubernetes with reasonable defaults. However, there are some settings that can be overridden using environment variables in [`controller.yaml`](../deploy/500-controller.yaml).
```

Comment:
This should be changed to reference the main branch - otherwise we end up with a broken link.